app/src/lib/trampoline/trampoline-environment.ts

```diff
@@ -134,9 +134,7 @@ export async function withTrampolineEnv<T>(
 
 const isAuthFailure = (e: unknown): e is GitError =>
   e instanceof GitError &&
-  (e.result.gitError === DugiteError.HTTPSAuthenticationFailed ||
-    // TODO: This should be in dugite instead of desktop!
-    /fatal: Authentication failed for 'http:/.test(e.result.stderr))
+  e.result.gitError === DugiteError.HTTPSAuthenticationFailed
 
```

Comment:
I built it locally and tested to double check this does in fact catch it via dugite like we tried in last beta.